lib/WP_Auth0_LoginManager.php

```diff
@@ -100,8 +101,27 @@ public function init_auth0() {
 
 		set_query_var( 'auth0_login_successful', false );
 
-		// Not an Auth0 login process or settings are not configured to allow logins.
+		$invitation   = $this->query_vars( 'invitation' );
+		$organization = $this->query_vars( 'organization' );
+
+		if ( $invitation && $organization ) {
+			$connection  = apply_filters( 'auth0_get_auto_login_connection', $this->a0_options->get( 'auto_login_method' ) );
+			$auth_params = self::get_authorize_params( $connection );
+
+			WP_Auth0_State_Handler::get_instance()->set_cookie( $auth_params['state'] );
+			WP_Auth0_Nonce_Handler::get_instance()->set_cookie( $auth_params['nonce'] );
+
+			$auth_params['invitation'] = $invitation;
+
+			$auth_url = self::build_authorize_url( $auth_params );
+
+			wp_safe_redirect( $auth_url );
```

Comment:
Correct! Since WordPress can support multiple virtual sites (WP MultiSite) on the same domain, we can't reliably tie the invitation flow to any specific endpoint, as customers might be using the same Auth0 Application to manage authentication across several sites. Since the 'Application Login URI' is static, handling this globally and letting the customer choose their invitation landing page made the most sense.

Reply:
Thanks!